polaris-core/src/main/java/org/apache/polaris/core/persistence/LocalPolarisMetaStoreManagerFactory.java

```diff
@@ -46,7 +46,6 @@
  */
 public abstract class LocalPolarisMetaStoreManagerFactory<StoreType>
     implements MetaStoreManagerFactory {
-
```

Comment:
nit: spurious change?

Reply:
Yeah.  Let me remove this unrelated change from this PR.